refiner-lib/src/utils.rs

```diff
@@ -5,6 +5,8 @@ use rlp::{DecoderError, Rlp};
 use sha3::Digest;
 use std::convert::TryFrom;
 
+const U64_MAX: U256 = U256([u64::MAX, 0, 0, 0]);
```

Comment:
I was little scary about this because of endianness but U256 doc says that it is little-endian so we should be always fine no matter the architecture.